CMakeLists.txt

```diff
@@ -195,7 +195,7 @@ target_include_directories(
 PUBLIC
     $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/include>
     $<INSTALL_INTERFACE:include>
-
+PRIVATE
     $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/lib/basisu/transcoder>
     $<INSTALL_INTERFACE:lib/basisu/transcoder>
```

Comment:
I wonder if this line and all similar are really necessary. I've requested a review from @atteneder.

Reply:
While this changes are not necessary, I agree it makes sense to strictly differentiate between public and private(internal) headers.

Reply:
I am referring to the `INSTALL_INTERFACE` line. lib/basisu/transcoder is not installed. It is purely internal to the libktx. Hence my question.

There shouldn't be any need to actually make and install a transcoder library. If shared libktx is being used I don't think anybody cares if the whole libktx is loaded into memory. If a static library is being used and only the transcoder is referenced, the linker will link only the transcoder functions into the application.